liblwgeom/liblwgeom.h.in

```diff
@@ -419,8 +426,8 @@ typedef struct
 	GBOX *bbox;
 	void *data;
 	int32_t srid;
+	lwflags_t flags;
 	uint8_t type;
-	uint8_t flags;
 	char pad[2]; /* Padding to 24 bytes (unused) */
```

Comment:
Small note: Please remember to reduce the pad from 2 extra bytes to just 1 the structs.

Reply:
thanks